src/OAuth2/Storage/Phalcon/Phalcon.php

```diff
@@ -664,4 +685,204 @@ public function getBuildSql($dbName = 'oauth2_server_php')
         return $sql;
     }
 
+}
+
+
+/**
+ * Class PhalconConf
+ * @package OAuth2\Storage\Phalcon
+ *
+ * This config class is used purely internally for communication
+ * between the model files and the main Phalcon storage class
+ * for setting table names dynamically.
+ */
+class PhalconConf
```

Comment:
I believe so. I'll branch and try it out. Would this be preferable? I always liked explicitly constructed models. I'll work it out and see if it works.